src/stores/courseData.js

```diff
@@ -260,7 +259,7 @@ const store = new Vuex.Store({
     },
     updateProgress (state, progress) {
       Vue.set(state.roads[state.activeRoad].contents.progressOverrides, progress.listID, progress.progress);
-      Vue.set(state.roads[state.activeRoad], 'changed', moment().format(DATE_FORMAT));
+      state.roads[state.activeRoad].changed = moment().format(DATE_FORMAT);
```

Comment:
I don't think I understand when set is and isn't necessary. how are these lines really different?

Reply:
They do the same thing.  The only place that you need a Vue.set is when you are adding a property to an object that wasn't there before.  i.e. if you are adding a new road to state.roads, you need to tell Vue about it because that key wasn't there before.  But here, the 'changed' key was already there and we are just updating it.  It's already reactive so we don't have to worry about Vue.sets

Reply:
Oh, I see. Thanks